site/app/views/NavigationView.php

```diff
@@ -653,6 +680,28 @@ public function showGradeables($sections_to_list) {
         $return .= <<<HTML
                             </table>
                         </div>
+HTML;
+        return $return;
+    }
+
+    public function deleteGradeableForm() {
+        $return = <<<HTML
+    <div class="popup-form" id="delete-gradeable-form" style="width:550px; margin-left:-250px;">
+        <h2>Delete Gradeable</h2> 
+        <p>&emsp;</p>
+        <p>Note: A Gradeable can only be deleted if it do not have any team and submission.
```

Comment:
Note: A gradeable can only be deleted if it has no student teams and if it has no student submissions.